pykickstart/handlers/f23.py

```diff
@@ -56,7 +56,7 @@ class F23Handler(BaseHandler):
         "lang": commands.lang.F19_Lang,
         "liveimg": commands.liveimg.F19_Liveimg,
         "logging": commands.logging.FC6_Logging,
-        "logvol": commands.logvol.F20_LogVol,
```

Comment:
*f21.py* has these set to ``F21_LogVol`` and ``F21_LogVolData``. But *f22.py* doesn't, so...oops! We should inspect these files and make sure the most recent versions are used in the future.